lib/lotus/router.rb

```diff
@@ -902,7 +902,7 @@ def call(env)
     #   router.path(:login, return_to: '/dashboard') # => "/login?return_to=%2Fdashboard"
     #   router.path(:framework, name: 'router')      # => "/router"
     def path(route, *args)
-      @router.path(route, *args)
+      @router.custom_path(route, *args)
```

Comment:
it's very weird but `path` didn't call :S and the custom exception didn't raise, that's why I fix exception tests.